src/tools/ApiChecker/ScaffoldTestOptions.cs

```diff
@@ -0,0 +1,22 @@
+using CommandLine;
+
+namespace ApiChecker
+{
+#pragma warning disable SA1600 // Elements are self-documenting and don't need additional docs
```

Comment:
I would add the comments and make the analyzer happy instead.

Reply:
As per our conversation, I've marked it `internal`